src/test_l1_hdmi_cec_driver.c

```diff
@@ -1135,7 +995,7 @@ void test_hdmicec_hal_l1_getLogicalAddress_sourceDevice_positive( void )
  */
 void test_hdmicec_hal_l1_setRxCallback_negative ( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```

Comment:
at line 1020, change the comment.

Reply:
@akhilbhas ,  This GetLogicalAddress is applicable for both source and sink devices.  This is common as the source devices gets their logical address from Open() and then provide the same through this API.

Reply:
updated the descriptions